ui/i18n/qml_en.ts

```diff
@@ -4,10 +4,10 @@
 <context>
     <name>AppMain</name>
     <message numerus="yes">
-        <location filename="../app/mainui/AppMain.qml" line="627"/>
-        <location filename="../app/mainui/AppMain.qml" line="636"/>
-        <location filename="../app/mainui/AppMain.qml" line="627"/>
-        <location filename="../app/mainui/AppMain.qml" line="636"/>
+        <location filename="../app/mainui/AppMain.qml" line="672"/>
```

Comment:
Do we have documentation on how and when to update the source file for the translations?
Is it a `lupdate` call only?

Reply:
It's a script which effectively calls lupdate

Reply:
ah, ok, found it, thank you